wayback/_client.py

```diff
@@ -400,17 +400,21 @@ def send(self, *args, **kwargs):
         retries = 0
         while True:
             try:
+                logger.debug("sending %s", kwargs)
                 result = super().send(*args, **kwargs)
                 if retries >= maximum or not self.should_retry(result):
                     if result.status_code == 429:
+                        logger.warning("caught rate limit error: %s", result.content)
                         raise RateLimitError(result)
                     return result
             except WaybackSession.handleable_errors as error:
+                logger.warn("caught exception during request: %s", error)
                 response = getattr(error, 'response', None)
                 if response:
                     read_and_close(response)
 
                 if retries >= maximum:
+                    logger.error("Too many retries %s >= %s", retries, maximum)
```

Comment:
Same as above here. This raises all the way up to user code, so I do not think logging, or at least not logging at a level above `info` at most, makes a lot of sense at this point.

Reply:
Ok, removing.